frida_mode/src/instrument/instrument_x64.c

```diff
@@ -216,6 +216,8 @@ static gboolean instrument_coverage_find_low(const GumRangeDetails *details,
   static GumAddress last_limit = (64ULL << 10);
```

Comment:
We start looking from 64kb as that is a conservative value for the size of the zero page.